spec/rubocop/cop/style/word_array_spec.rb

```diff
@@ -101,6 +101,11 @@
     expect(new_source).to eq('%W(one two \n \t)')
   end
 
+  it 'keeps the line breaks in place after auto-correct' do
+    new_source = autocorrect_source(cop, "['one',\n 'two', 'three']")
```

Comment:
The second argument should be an array of lines instead of including the line break in a single string.